spec/symengine_spec.rb

```diff
@@ -18,5 +18,16 @@
     it { is_expected.to be_a SymEngine::Rational }
     its(:to_s) { is_expected.to eq '1/3' }
   end
+  
+  describe 'parse' do
+    subject { SymEngine::parse('123 + 321') }
```

Comment:
Since these specs act as documentation, can you add more strings for parsing?
eg.

- `x + (y * 3 - 5)`
- `x ** y + f(x)`
- `sin(x) + cos(y) / 2`
- `1 / 2 + 3 / 2`
- `1 + 2 * 2`
- `tan(0) - sqrt(2)`